env/local.sh.example

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+export DATABASE_URL=postgres://nbap@localhost/niq_careers
```

Comment:
Use same parameters of postres container in `devops/base_files/docker-compose.base` 
```BASH
export DATABASE_URL=postgres://niq_user:niq_pass@postgres_niq_careers/niq_db
```